boards/silabs/radio_boards/slwrb4170a/slwrb4170a.dts

```diff
@@ -6,11 +6,11 @@
 
 /dts-v1/;
 #include <silabs/efr32mg12p433f1024gm68.dtsi>
-#include "efr32_radio.dtsi"
+#include "../efr32-series1-common.dtsi"
```

Comment:
create a common folder for these files, do not litter the main board directory

Reply:
Done!